backend/src/app/services/site/site.service.ts

```diff
@@ -309,28 +313,52 @@ export class SiteService {
    * @param siteId site Id
    * @returns a single site matching the site ID
    */
-  async findSiteBySiteId(siteId: string, pending: boolean) {
+  async findSiteBySiteId(siteId: string, pending: boolean, userInfo: any) {
     this.sitesLogger.log('SiteService.findSiteBySiteId() start');
     this.sitesLogger.debug('SiteService.findSiteBySiteId() start');
     const response = new FetchSiteDetail();
 
     response.httpStatusCode = 200;
 
-    if (pending) {
-      const result = await this.siteRepository.findOne({
-        where: { id: siteId, srAction: SRApprovalStatusEnum.PENDING },
-      });
+    let snapShot: Snapshots = null;
 
-      response.data = result ? result : null;
+    if (!userInfo) {
+      this.sitesLogger.debug(
+        'SiteService.findSiteBySiteId() user not logged in',
+      );
+
+      snapShot = null;
+    } else if (userInfo?.identity_provider === 'idir') {
+      this.sitesLogger.debug(
```

Comment:
do you require info or debug logs here ?

Reply:
updated to use 'log'